auth/README.md

```diff
@@ -0,0 +1,76 @@
+# Go client library for signing in user
+
+Go client library for signing in user.
+
+## Install
+
+```bash
+go get github.com/canopas/apple-sdk-go/auth
+```
+
+## How to use?
+
+- **TeamId** :  10-char App Id prefix found in App identifiers section (Ex: AB65CD4321)
+
+- **ClientId** : ClientID is the "Services ID" value that you get when navigating to your "sign in with Apple"-enabled service ID (Ex: com.example.me)
+
+- **KeyId** : This is the ID of the private key (Ex: FE12DC34BA)
+
+- **Secret** : This is the private key file (.p8). You can download it from [apple portal](https://developer.apple.com/account/resources/)
+
+
+```go
+
+req := auth.New("team-id", "client-id", "key-id", "secret-key")
+
+resp, err := req.ValidateCode("auth-code") 
+
+// or
+
+resp, err := req.ValidateCodeWithRedirectURI("auth-code", "redirect-uri") 
+
+// or
+
+resp, err := req.ValidateRefreshToken("refresh-token") 
+
+if err != nil {
+	log.Fatal(err.Error())
+}
+
+// get validated user
+user, err := resp.GetUser(resp.IDToken)
```

Comment:
Is it validated user or verified or authorised user?

Reply:
We get users from validation, that's why it's validated. Let's call it simply `user`